experiments/slack-events-api-test/Message.py

```diff
@@ -0,0 +1,17 @@
+
+class Message():
+    def __init__(self, timestamp):
+        self.timestamp = timestamp
+        self.message_content = ''
+        self.message_sender  = ''
+        self.message_recipient = ''
+
+    def set_message_content(self, message_sender, message_recipient, message_content):
+        self.message_content = message_content
+        self.message_sender  = message_sender
+        self.message_recipient = message_recipient
+
+    def print_message(self):
+        print('''Message:\n \
```

Comment:
> Triple quotes interpret all the whitespace inside them, so i think this will print as double new lines.
It doesn't actually. I also thought this but then had to add the `\n`